crates/theme/src/registry.rs

```diff
@@ -26,40 +30,41 @@ pub struct ThemeMeta {
 ///
 /// This should not be exposed outside of this module.
 #[derive(Default, Deref, DerefMut)]
-struct GlobalThemeRegistry(ThemeRegistry);
+struct GlobalThemeRegistry(Arc<ThemeRegistry>);
```

Comment:
We're now storing the `ThemeRegistry` in an `Arc` so that it is easier to use in a background task.